storage/dvo_recommendations_storage.go

```diff
@@ -56,6 +56,7 @@ type DVORecommendationsStorage interface {
 		string,
 	) (types.DVOReport, error)
 	DeleteReportsForOrg(orgID types.OrgID) error
+	WriteHeartbeat(string, time.Time) error
```

Comment:
this is not called outside of the tests. Is it intended? I guess for now it is

Reply:
Yes for now we do not have modified the writers to start writing heartbeats data.